coral-schema/src/main/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverter.java

```diff
@@ -84,12 +84,58 @@ public Schema toAvroSchema(String dbName, String tableOrViewName, boolean strict
    * @return avro schema for a given Dali view [dbName, viewName]
    */
   public Schema toAvroSchema(String dbName, String tableOrViewName) {
```

Comment:
Please keep this method unchanged.

Reply:
should we not duplicate codes? Just curious. I can revert that.

Reply:
reverted.

Reply:
Thanks. I think this change does not belong to this PR. We could revisit the improvement later.